shai/src/main/protobuf/cmdExecutor.proto

```diff
@@ -10,4 +10,26 @@ option objc_class_prefix = "TEP";
 package shai;
 
 service CmdExecutor {
+  rpc run(CmdRequest) returns (CmdResponse);
+}
+
+message CmdRequest {
+  Cmd cmd = 1;
+
+  string config = 2;
```

Comment:
Does this also take serialized JSON, like `result` and `failure` below?
Maybe we should add a comment.